content/en/guides/models/app/settings-page/user-settings.md

```diff
@@ -33,16 +33,35 @@ To add a follow badge for the `@weights_biases` account on X, you could add a Ma
 ```
 In an `<img>` tag, you can specify `width`, `height`, or both. If you specify only one of them, the image's proportions are maintained.
 
-## Teams
+## Default team
+If you are a member of more than one team, the **Default team** section allows you to configure the default team to use when a run or a Weave trace does not specify a team. If you are a member of only one team, that team is the default and this section does not appear.
+
+Select a tab to continue.
```

Comment:
What do the tabs denote in the workflow? Are these types of accounts that customers can have? I legit don't know.

Reply:
Yep, exactly! In cloud, we have Multi-tenant SaaS and Dedicated Cloud deployment types, and we also have Self-Managed, which is deployed using Kubernetes in the customer's cloud tenant or datacenter. Dedicated Cloud is actually Self-Managed managed by W&B.